### ISSUE TITLE
UTXO tab didn't update after mix sucess 

### ISSUE THREAD

**Issue by hur4hur5** [sparrowwallet/sparrow#512]
I'm using whirlpool. This has been working without issue so far. But I've recently had a UTXO on the UTXO tab of my postmix account appear stuck. By this I mean that the tooltip on the row in question is still displaying the "Waiting for broadcast of 'txid...:index'" message and the blue bar graphic is swooshing side to side. 

I know that the transaction has actually been broadcast because I can find the address the bitcoin was in as a result of this older mix, search for it in a block explorer and see it has been part of two coinjoins. The first coinjoin was this 'older mix' and the address in question was an output and the txid...:index of this coinjoin is the one showing in the output column of the UTXO tab. In the second more recent coinjoin the address was an input whose txid...:index matches the one detailed in the tooltip. This second coinjoin is part of a mined block, so the transaction was definitely broadcast.

![Screenshot from 2022-04-09 11-51-10](https://user-images.githubusercontent.com/102880838/162569077-989f307e-a485-40d6-bb4e-42f2c7306708.png)

When I click the magnifying glass for this UTXO the opened transaction tab shows a pulsing paper aeroplane which I guess is part of the same messaging about awaiting broadcast.

Also worth noting as an aside (as this could be a UX improvement), when I select the the 'stuck' UTXO row the "Send Selected" button at the bottom is greyed out (which seems correct) but the send paper aeroplane icon next to the magnifying glass on the row in the table is still present and can be clicked to begin creating a transaction.

Anyway...

I then clicked the stop mixing button. Another UXTO which was a registered input to a mix lost it's tooltip and graphic as expected. But the 'stuck' one continued showing the out of date tooltip and graphic.

So then as a next step I closed Sparrow and reopened it. The wallet had a red icon showing which, gauging by the transactions tab means that it failed to subscribe to paths (which I guess means certain derivation paths). So I refreshed the wallet and the new UTXO appeared and the old address is as expected empty when I look at that address in the address tab.

TLDR: Sometimes Sparrow doesn't successfully update the UTXO tab after a mix has been successful.

**Comment by craigraw** [comment]
All this is caused by your Electrs (I am assuming here) not functioning properly. Unfortunately Sparrow must depend on the server for it's information of what's going on on the blockchain, and when Electrs fails to notify of incoming transactions Sparrow cannot update. I have introduced many workarounds to account for poor Electrs behaviour, but ultimately this can only go so far. The answer again is to switch to Fulcrum.
